src/config/software-documentation/index.md

```diff
@@ -0,0 +1,13 @@
+# Software documentation
+
+Most common format of documentation in Void Linux are [manual pages](./man.md).
```

Comment:
```suggestion
The most common format of documentation in Void Linux are [manual pages](./man.md).
```

I think `is` would be more correct, but I don't know for sure.

Reply:
> I think `is` would be more correct

Any native speaker here?

Reply:
The subject is "the most common format", so "is" would be right here, but it's still a bit awkward. I suggest one of the below alternatives:

> The most common format for documentation is the `[manual page](./man.md)`.

> The most common media for documentation are `[manual pages](./man.md)`.

Reply:
"Media" suggestion is neat, thanks.